tests/test_core/test_alignment.py

```diff
@@ -390,7 +390,7 @@ def test_get_identical_sets(self):  # ported
         got = frozenset(frozenset(s) for s in got)
         self.assertEqual(got, expect)
 
-    def test_is_ragged(self):  # ported
+    def test_is_ragged(self):  # will not port
```

Comment:
why not porting this? (message me in Zulip)

Reply:
messaged